teachings.json

```diff
@@ -900,10 +900,131 @@
   {
     "name": "Computer Vision",
     "url": "computer-vision",
-    "chat": "",
+    "chat": "https://t.me/+9wTytT_D4Lg1MjFk",
     "website": "479049",
     "professors": [
       "giuseppe.lisanti"
     ]
+  },
+  {
+    "name": "Artificial Intelligence, Blockchain e criptovalute nello sviluppo software",
+    "url": "artificial-intelligence-blockchain-e-criptovalute-nello-sviluppo-software",
+    "chat": "https://t.me/+FrU7IhMlaZoyZDZk",
+    "website": "495981",
+    "professors": [
+      "giancarlo.succi"
+    ]
+  },
+  {
+    "name": "Cryptography",
+    "url": "cryptography",
+    "chat": "https://t.me/+h0CF62a0rNRiNDQ0",
+    "website": "479060",
+    "professors": [
+      "ugo.dallago"
+    ]
+  },
+  {
+    "name": "Scalable and Cloud Programming",
+    "url": "scalable-and-cloud-programming",
+    "chat": "https://t.me/+-7R_uvQAWGhiODZk",
+    "website": "479058",
+    "professors": [
+      "gianluigi.zavattaro"
+    ]
+  },
+  {
+    "name": "Business Intelligence",
+    "url": "business-intelligence",
+    "chat": "https://t.me/+zoNiPXUXuOFhYjE0",
+    "website": "479052",
+    "professors": [
+      "furio.camillo"
+    ]
+  },
+  {
+    "name": "Computational Imaging",
+    "url": "computational-imaging",
+    "chat": "https://t.me/+1cVwLP7gVK9hMjg8",
+    "website": "503547",
+    "professors": [
+      "elena.lolipiccolomini",
+      "davide.evangelista"
+    ]
+  },
+  {
+    "name": "Multimedia Data Management",
+    "url": "multimedia-data-management",
+    "chat": "https://t.me/+eX3Fjlbq90MzMzU0",
+    "website": "479051",
+    "professors": [
+      "ilaria.bartolini"
+    ]
+  },
+  {
+    "name": "Archietture Software a Microservizi",
+    "url": "architetture-software-a-microservizi",
+    "chat": "https://t.me/+5NgnXeO4C5Q2Y2Zk",
+    "website": "503417",
+    "professors": [
+      "davide.rossi",
+      "ivan.lanese"
+    ]
+  },
+  {
+    "name": "aboratorio di Making",

```

Comment:
typo: "laboratorio" instead of "aboratorio"

Reply:
Fixed in 4f95e22